account-kit/core/src/store/store.ts

```diff
@@ -218,12 +231,15 @@ export const createSigner = (params: ClientStoreConfig) => {
  * Converts the AlchemySigner's status to a more readable object
  *
  * @param {AlchemySignerStatus} alchemySignerStatus Enum value of the AlchemySigner's status to convert
+ * @param {ErrorInfo | undefined} error the current signer error, if present
  * @returns {SignerStatus} an object containing the original status as well as booleans to check the current state
  */
 export const convertSignerStatusToState = (
-  alchemySignerStatus: AlchemySignerStatus
+  alchemySignerStatus: AlchemySignerStatus,
+  error: ErrorInfo | undefined
```

Comment:
The way this function is being used, it would most likely be a mistake to omit the error parameter. Passing `undefined` here clears the error state, which should be a deliberate choice and not the default behavior of leaving out a parameter.

Reply:
Yeah that is very fair.